client/src/nl/sensorlab/videowall/animation/baseanimations/bouncypixels/BouncyPixelSystem.java

```diff
@@ -0,0 +1,59 @@
+package nl.sensorlab.videowall.animation.baseanimations.bouncypixels;
+
+import java.util.ArrayList;
+
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import processing.core.PGraphics;
+
+public class BouncyPixelSystem {
+	
+	BouncyPixelsAnimation parent;
+	
+	public ArrayList<BouncyPixel> bouncypixels;
+	private int[] colors;
+	
+	public float spring;
+	public float gravity;
+	public float friction;
+	
+	public BouncyPixelSystem(BouncyPixelsAnimation parent, int[] colors, float spring, float gravity, float friction) {
+		this.parent = parent;
+		this.colors = colors;
+		this.spring = spring;
+		this.gravity = gravity;
+		this.friction = friction;
+		this.bouncypixels = new ArrayList<BouncyPixel>();
+	}
+	
+	public void generateBouncyPixels(int amountBouncyPixels) {
+		for(int i = 0; i < amountBouncyPixels; i++) {
+			addBouncyPixel();
+		}
+	}
+	
+	public void addBouncyPixel() { 
+		float x = (float)(Math.random() * BaseAnimation.PIXEL_RESOLUTION_X);
+		float y = -(float)(Math.random() * BaseAnimation.PIXEL_RESOLUTION_Y);// Start outside window height
+		float diameter = 1; // <-- Because it are pixels we want this to be set to 1
```

Comment:
If this is fixed (static) why isn't it a constant? Then it doesn't need to be in memory and not passed around in methods.